screens/SingleDayScreen.tsx

```diff
@@ -49,6 +50,22 @@ const SingleDayScreen: React.FC<
     getForecast,
   } = useWeatherData(city, today);
 
+  const setAnimationState = (isRunning: boolean) => {
+    setIsAnimationRunning(isRunning);
+  };
```

Comment:
Nie wydaje się, żeby to było do czegokolwiek potrzebne - ta funkcja nie robi nic poza wywołaniem tego settera state'u

Reply:
ten kod rowniez